lib/log.js

```diff
@@ -208,46 +225,124 @@ const util = require('util'),
+                makeSession(options, next);
+            }
 
-                if (Array.isArray(str.match(exp))) {
-                    str.match(exp).forEach(function(item) {
-                        console.log(item.trim()); // Do not remove
+            function _checkUser(next) {
+                checkUser(options, next);
+            }
+        },
+
+        checkLogDaemon: function (options, next) {
+            log.info("log#checkLogDaemon()");
+            if (typeof next !== 'function') {
+                throw errHndl.getErrMsg("MISSING_CALLBACK", "next", util.inspect(next));
+            }
+
+            options = options || {};
+
+            let serviceParam = "",
+                cmd = "",
+                logFilePath = "",
+                returnData = "",
+                logDaemonStatus = 0,
+                existLogFile = 0;
+
+            async.series([
+                _makeSession,
+                _setparam,
+                function(next) {
+                    options.session.run(cmd, process.stdin, _checkService, process.stderr, function(err) {
+                        if (err) {
+                            return next(err);
+                        }
+                        if (!logDaemonStatus) {
+                            return next(errHndl.getErrMsg("NOT_MATCHED_LOGDAEMON", options.currentDaemon === "journald" ? "pmlogd" : "journald"));
+                        }
+
+                        cmd = `ls -al ${logFilePath}`;
+                        options.session.run(cmd, process.stdin, _checkFile, process.stderr, function(error) {
+                            if (error) {
+                                return next(err);
+                            }
+                            if (!existLogFile) {
+                                return next(errHndl.getErrMsg("NOT_EXIST_LOGFILE"));
+                            }
+                            next();
+                        });
                     });
+                },
+            ], function(err) {
+                if (err) {
+                    console.log("CLI's current log daemon : " + options.currentDaemon);
+                    next(err, null);
+                } else {
+                    next(null, {msg : "CLI's current log daemon : " + options.currentDaemon
+                                        + "\nThe target's current log daemon : " + options.currentDaemon});
+                }
+            });
+
+            function _setparam(next) {
+                log.info("log#checkLogDaemon()#_setparam()");
+                if (options.currentDaemon === "journald") {
+                    serviceParam = "systemd-journald";
+                    logFilePath = "/run/log/journal";
+                } else if (options.currentDaemon === "pmlogd") {
+                    serviceParam = "pm-log-daemon";
+                    logFilePath = "/var/log/messages";
                 }
+
+                cmd = `systemctl status ${serviceParam}.service | tee`;
```

Comment:
Could you check the difference betwwen /usr/bin/systemctl and systemctl?
CLI calls shell command "/usr/bin/opkg, /bin/cat instead of opkg, cat.

Reply:
They are same. 
systemctl is a command and if systemctl is called, actually /usr/bin/systemctl is run.
It is not different linux pc and same as 'journalctl' or 'tail'

Reply:
okay.